gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/task/EhrExtractMessageHandlerTest.java

```diff
@@ -248,23 +243,20 @@ public void When_HandleMessageWithValidDataIsCalled_Expect_CallSendContinueReque
 
         inboundMessage.setPayload(readLargeInboundMessagePayloadFromFile());
         inboundMessage.setEbXML(readLargeInboundMessageEbXmlFromFile());
+        List<InboundMessage.ExternalAttachment> externalAttachmentsTestList = new ArrayList<>();
+        externalAttachmentsTestList.add(
```

Comment:
this could do with being pulled into a priv set up method since it gets repeated a few times & interferes with the readability of test.
i think i'd put a "count" input to this method so you can add as many as you need, with default of one?